packages/nimble-components/src/table-column/base/tests/table-cell-view.spec.ts

```diff
@@ -16,6 +18,24 @@ describe('TableCellView', () => {
     let connect: () => Promise<void>;
     let disconnect: () => Promise<void>;
 
+    const tableColumnDelegatesClickAndKeydownTag = 'nimble-test-table-column-delegates';
```

Comment:
Since this is not globally shared use the `uniqueElementName` pattern that was linked to: https://github.com/ni/nimble/pull/1246#discussion_r1198028651

Reply:
Done